python/pyspark/sql/catalog.py

```diff
@@ -185,6 +185,14 @@ def tableExists(self, tableName, dbName=None):
         >>> df = spark.sql("CREATE TABLE tab1 (name STRING, age INT) USING parquet")
         >>> spark.catalog.tableExists("tab1")
         True
+        >>> spark.catalog.tableExists("view1")
```

Comment:
Since we're improving the doc tests here, shall we make it more readable? e.g.)

```
Examples
----------

The example below checks if 'unexisting_table' table.

>>> spark.sql("CREATE TABLE tab1 (name STRING, age INT) USING parquet")
>>> spark.catalog.tableExists("tab1")
True
>>> spark.sql("DROP TABLE tab1")
>>> spark.catalog.tableExists("unexisting_table")
False

This function can check if a view is defined or not:

>>> df = spark.sql("CREATE VIEW view1 AS SELECT * FROM tab1")
>>> spark.catalog.tableExists("view1")
True
>>> df = spark.sql("DROP VIEW view1")
>>> spark.catalog.tableExists("view1")
False
```

Reply:
Thanks for the comment. Changed